src/sidebar/services/annotations-exporter.tsx

```diff
@@ -152,6 +152,89 @@ export class AnnotationsExporter {
     return `${headers}\n${annotationsContent}`;
   }
 
+  buildHTMLExportContent(
+    annotations: APIAnnotationData[],
+    {
+      groupName = '',
+      displayNamesEnabled = false,
+      defaultAuthority = '',
+      /* istanbul ignore next - test seam */
+      now = new Date(),
+    }: ExportOptions = {},
+  ): string {
+    const { uri, title, uniqueUsers, replies, extractUsername } =
+      this._exportCommon(annotations, {
+        displayNamesEnabled,
+        defaultAuthority,
+      });
+
+    return renderToString(
+      <html lang="en">
+        <head>
+          <title>Annotations export - Hypothesis</title>
+          <meta charSet="UTF-8" />
+        </head>
+        <body>
+          <section>
+            <h1>Summary</h1>
+            <p>
+              <time dateTime={now.toISOString()}>{formatDateTime(now)}</time>
+            </p>
+            <p>
+              <strong>{title}</strong>
+            </p>
+            <p>
+              <a href={uri} target="_blank" rel="noopener noreferrer">
+                {uri}
+              </a>
+            </p>
+
+            <dl>
+              <dt>Group</dt>
+              <dd>{groupName}</dd>
+              <dt>Total users</dt>
+              <dd>{uniqueUsers.length}</dd>
+              <dt>Users</dt>
+              <dd>{uniqueUsers.join(', ')}</dd>
+              <dt>Total annotations</dt>
+              <dd>{annotations.length}</dd>
+              <dt>Total replies</dt>
+              <dd>{replies.length}</dd>
+            </dl>
+          </section>
+          <hr />
+          <section>
+            <h1>Annotations</h1>
+            {annotations.map((annotation, index) => {
+              const page = pageLabel(annotation);
+              return (
+                <article key={annotation.id}>
+                  <h2>Annotation {index + 1}:</h2>
+                  <p>
+                    <time dateTime={annotation.created}>
+                      {formatDateTime(new Date(annotation.created))}
+                    </time>
+                  </p>
+                  <p>Comment: {annotation.text}</p>
+                  <p>{extractUsername(annotation)}</p>
+                  <p>
+                    Quote: <blockquote>{quote(annotation)}</blockquote>
+                  </p>
```

Comment:
I would prefer that we order the content here roughly in the same way that we present it on annotation cards: Metadata (author, timestamp), then quote, then comment, then tags.

Reply:
Yeah, there's another item to address that, both for HTML and Text.